File: addOns/jython/src/main/zapHomeFiles/scripts/templates/targeted/Targeted default template.py
"""
Targeted scripts can only be invoked by you, the user, e.g. via a right-click option on the Sites or History tabs
"""

def invokeWith(msg):
  # Debugging can be done using print like this
  print('invokeWith called for url=' + msg.getRequestHeader().getURI().toString()); 

File: addOns/jruby/src/main/zapHomeFiles/scripts/templates/targeted/Targeted default template.rb
# Targeted scripts can only be invoked by you, the user, e.g. via a right-click option on the Sites or History tabs

def invokeWith(msg)
  # Debugging can be done using puts like this
  puts('invokeWith called for url=' + msg.getRequestHeader().getURI().toString()); 
end